extension.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<software>
+    <metadata>
+        <name>Batchtool (FORK)</name>
+        <version>1.3</version>
+        <copyright>Copyright ɠ2010 Arne Bakkebøy</copyright>
```

Comment:
Thank you, I certainly will. :) Could you remove the y in Bakkebø first, and the "(FORK)" in name, please? And the version here should be 1.1, the same as in composer.json.